Tests/src/PublicAPITests/UserSystemTests.cpp

```diff
@@ -80,12 +80,16 @@ void LogIn(csp::systems::UserSystem* UserSystem,
 		   csp::common::String& OutUserId,
 		   const csp::common::String& Email,
 		   const csp::common::String& Password,
-		   csp::services::EResultCode ExpectedResultCode)
+		   bool AgeVerified,
+		   csp::services::EResultCode ExpectedResultCode,
+		   csp::services::EResultBaseFailureReason ExpectedResultFailureCode)
 {
-	auto [Result] = Awaitable(&csp::systems::UserSystem::Login, UserSystem, "", Email, Password, nullptr).Await(RequestPredicate);
+	auto [Result] = Awaitable(&csp::systems::UserSystem::Login, UserSystem, "", Email, Password, AgeVerified).Await(RequestPredicate);
 
 	EXPECT_EQ(Result.GetResultCode(), ExpectedResultCode);
 
+	EXPECT_EQ(Result.GetFailureReason(), int(ExpectedResultFailureCode));
```

Comment:
Prefer `static_cast<int>` over C-style casts, or at least use `(int)` instead of `int(x)`

Reply:
done